partiql-conformance-tests/tests/mod.rs

```diff
@@ -107,18 +117,24 @@ pub(crate) fn fail_eval(statement: &str, mode: EvaluationMode, env: &Option<Test
 
     let parsed = parse(statement);
     let lowered_result = lower(&catalog, &parsed.expect("parse"));
-    let lowered = lowered_result.expect("lower");
+    let lowered = lowered_result.expect("blah blah blah lower");
```

Comment:
Probably want to remove "blah blah blah"?

Reply:
```suggestion
    let lowered = lowered_result.expect("lower");
```